client/src/ui/components/Tutorial/TutorialBlock.tsx

```
@@ -0,0 +1,105 @@
+  isHighlighted,
+  highlightType = "block",
+  isClickable,
+}) => {
+  const ref = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    if (ref.current === null) return;
+
+    const element = ref.current; // Capture the current value
+    const onTransitionStart = () => {
+      onTransitionBlockStart && onTransitionBlockStart();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)